### PR TITLE
Message persistent with localStorage

### PR DESCRIPTION
**Pull request by moo-light**
Main description: added localstorage messagePersistent when user lost connection or reloading the page
 
added `messageLog` language change when user change to language
added `storage` and `initStorage` to retain data for **questioner** and **responder**
added `.prettierrc` for easier code format

updated `pubnub` message receiving channel to save message to local storage
updated `lastMessageTimestamp` will be set to `0` when different **role** send a message

Removed `updateShareLink()` function to retain the sharelink `chain icon`🔗 when changing languages 

> ✔️ most changes are commented in code source

### CODE REVIEW

**Comment by bacloud22** [comment]
Hi @moo-light I find it hard to follow with the new indentation. 
It is definitely serious work so thank you so much 🙏 

Could you please try to apply the earlier indentation (tweaking prettierrc config ?)

**Comment by bacloud22** [comment]
Impressive work! 👏

**Comment by moo-light** [comment]
> Hi @moo-light I find it hard to follow with the new indentation. It is definitely serious work so thank you so much 🙏
> 
> Could you please try to apply the earlier indentation (tweaking prettierrc config ?)

okay i will change the tabs indentation from 4 to 2

**Review by bacloud22 — APPROVED** [review]
